x-pack/legacy/plugins/apm/server/lib/helpers/es_client.ts

```diff
@@ -15,22 +15,12 @@ import {
 import { Legacy } from 'kibana';
 import { cloneDeep, has, isString, set } from 'lodash';
 import { OBSERVER_VERSION_MAJOR } from '../../../common/elasticsearch_fieldnames';
-import { StringMap } from '../../../typings/common';
+import { StringMap, Omit } from '../../../typings/common';
+import { getApmIndices } from '../settings/ui_indices/apm_ui_indices';
```

Comment:
naming nit: is it "apm indices", "apm ui indices" or "ui indices"? This line has all three of them :D
Personally I think "apm indices" is the most fitting. Because the indices are not necessarily tied to the UI.

Btw. we can label this differently to the end user if we think - this is just about what we call it under the hood.

Reply:
i made all the naming consisten with "apmIndices"